doc_source/reference.md

```diff
@@ -4,9 +4,36 @@ The [API Reference](https://docs.aws.amazon.com/cdk/api/latest) contains informa
 
 Each library contains information about how to use the library\. For example, the [S3](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-s3-readme.html) library demonstrates how to set default encryption on an Amazon S3 bucket\.
 
-## Versioning and Stability Model<a name="versioning"></a>
+## Versioning<a name="versioning"></a>
 
-Version numbers consist of three numeric version parts: *major*\.*minor*\.*patch*, and adhere to the [semantic versioning](https://semver.org) model\. This means that breaking changes to stable APIs, which is described in the next section, are limited to major releases\. Minor and patch releases are backward compatible, meaning that the code written in a previous version with the same major version can be upgraded to a newer version and be expected to continue to build and run, producing the same output\.
+Version numbers consist of three numeric version parts: *major*\.*minor*\.*patch*, and adhere to the [semantic versioning](https://semver.org) model\. This means that breaking changes to stable APIs, are limited to major releases\. Minor and patch releases are backward compatible, meaning that the code written in a previous version with the same major version can be upgraded to a newer version and be expected to continue to build and run, producing the same output\.
+
+## AWS CDK CLI Compatibility<a name="aws_cdk_compatibility"></a>
+
+Following is the expected behavior of the CLI in relation to different versions of the framework libraries.
```

Comment:
The terminology "framework libraries" in not that clear. We should explain in more detail (and maybe with an example) the relationship between the CLI version and the versions used by the user's CDK application (i.e. "framework libraries")

Reply:
what if we just use `construct libraries`?